src/mutation-observer.js

```diff
@@ -100,13 +100,13 @@ MutationObserver.fixIe = function () {
   }
 
   // We have to call the old innerHTML getter and setter.
-  var oldInnerHtml = Object.getOwnPropertyDescriptor(elementPrototype, 'innerHTML');
```

Comment:
Yeah, I'm breaking the style guide in this instance because I think consistency with `innerHTML` is more important: I think it makes it easier to make the mental connection of what the variable is.